clients/egui/src/main.rs

```diff
@@ -23,7 +23,7 @@ fn main() {
     // following issues:
     //  1. window decorations are non-native.
     //  2. dragging & dropping from the system doesn't work.
-    std::env::set_var("WINIT_UNIX_BACKEND", "x11");
+    std::env::set_var("WINIT_UNIX_BACKEND", "wayland");
```

Comment:
?

Reply:
oops, i'm gonna do what you said in standup and only hard set that var if it's not defined by the user. it's getting a bit annoying.